Sources/Conformance/conformance.pb.swift

```diff
@@ -180,18 +180,18 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message, SwiftProtobuf.Prot
       self = .None
     }
 
-    public mutating func decodeField<T: SwiftProtobuf.FieldDecoder>(setter: inout T, protoFieldNumber: Int) throws {
-      if self != .None && setter.rejectConflictingOneof {
-        throw SwiftProtobuf.DecodingError.duplicatedOneOf
+    public mutating func decodeField<T: SwiftProtobuf.Decoder>(decoder: inout T, fieldNumber: Int) throws {
+      if self != .None {
+        try decoder.handleConflictingOneOf()
```

Comment:
Since there's no longer a single `DecodingError`, I wanted to provide a way for individual decoders to throw appropriate errors.  So I changed the boolean property `rejectConflictingOneof` to a method that is a no-op for decoders that don't want to reject conflicts here.